task/04-arrays-tasks.js

```diff
@@ -536,7 +614,9 @@ function selectMany(arr, childrenSelector) {
  *   [[[ 1, 2, 3]]], [ 0, 0, 1 ]      => 2        (arr[0][0][1])
  */
 function getElementByIndexes(arr, indexes) {
-    throw new Error('Not implemented');
+  indexes.map(x => arr = arr[x]);
```

Comment:
do not modifiy arguments if not required.

Result of map funcitons is not used